### ISSUE TITLE
App approval

### ISSUE THREAD

**Issue by sDamme** [dejanstojanovic/Facebook-Album-Browser#29]
Hello,
It might be my own misunderstanding or something else but let's say I am an admin of a tattoo page and I would want to get images from that page to a website, in that case do I need to get my app approved for it to work correctly or no?
Currently I have followed the instructions and got far enough to get the name and profile picture of that page to show but it wont load the images, the loader icon just keeps going and going. I don't see any errors as well.
Seeing as it does show the name and profile picture of the page, I am quessing that everything should be fine with my configuration and thus must be something else that keeps the images from loading, as in the app needs to be approved, right?
Or perhaps there is something wrong with my app permissions? I am not sure. :(

Apologies for the long post and my own misunderstanding. I can also post a link to the site if you think it is needed.
Thanks in advance.

**Comment by dejanstojanovic** [comment]
Hi,
For pages you do not need any app approval from Facebook. You just need to create an app and token for it which you will use for the plugin.
Can you give me the URL of the page where you are using the plugin? I would like to have a look what might went wrong if possible.

**Comment by sDamme** [comment]
Sure, it's sander2.alimit.com/assets/image-get.html 
I hope you can see something that I might have missed. :)

**Comment by comtronix** [comment]
the Graph is empty.
https://graph.facebook.com/TartooTattoo?access_token=520461818121056|bS2WoBlm1P7FZUQspdelAUJZh2c&callback=jQuery111008208020077296023_1440397777942&_=1440397777943

i think your accestoken is not valid.

**Comment by sDamme** [comment]
I see, I will try to generate a token again, maybe I missed something in the guide on how to make it. Thank you either way. I'll report back if I get any luck or no.

**Comment by sDamme** [comment]
I suspect that if I am not the admin of the app then my access token is invalid?

 So if I was assigned as a developer for the app then it is not going to work and I need to get admin access to the app? 
Other than that I have no clue.
I'll try to get a hang of admin account or permission. Then see where that leads me. Facebook really likes to make things difficult. :P

**Comment by dejanstojanovic** [comment]
Well, yes, it was a lot easier before they introduced these access tokens :)
I think you need to be the owner of the app, for example token in the demo is generated for my app I created for that purpose in Facebook developer section. However the page does not belong to me but it is still working.
Login to your Facebook account, signup as a developer, create an app and then use it's token. Should work, at least it worked for me

**Comment by sDamme** [comment]
Wow, this is really weird. I did as you suggested. Made a new app and everything. Got a new token as well. Tried with it, nothing.
I then thought what the hell. No way it would work with the token in your demo, but what ya know, it worked with it...So it's a win I guess.
Still I have no idea why it will not work with my tokens. And I am not sure if this is really the end-solution to this issue as well. I would still need a non-expiring token for it.

Any thoughts, comments, suggestions?
